### PR TITLE
Pull request/2558a0bf

### PR DESCRIPTION
**Pull request by bpostlethwaite**
Closing JuliaLang/METADATA.jl/pull/2050

### CODE REVIEW

**Comment by ViralBShah** [comment]
The package does need to have .jl in its name.

**Comment by bpostlethwaite** [comment]
@ViralBShah How do I rename the package?

**Comment by ViralBShah** [comment]
I assume github will allow renaming your project. You probably want to call it `Plotly.jl`. Doing that and then updating the URL in this PR should hopefully suffice - or create a new package with that name.

**Comment by milktrader** [comment]
The package has been renamed but for some reason we still have the url to the old name. I would just manually change the name

**Comment by bpostlethwaite** [comment]
Ah, I see what is happening now :) 
I gotta say, at first this system can be a little confusing.

**Comment by milktrader** [comment]
Thanks for your patience and stress testing the system for us :smile:

**Comment by milktrader** [comment]
I'm not sure why this is showing up as Plotly.jl in the METADATA. @ViralBShah @IainNZ

File: Plotly.jl/url [hunk 1]
```diff
@@ -0,0 +1 @@
+git://github.com/plotly/Plotly.jl.git
```
Comment (bpostlethwaite) [inline Plotly.jl/url]: it is now...

Comment (milktrader) [inline Plotly.jl/url]: Yeah, I noticed that got fixed but your folder in METADATA shows Plotly.jl versus just Plotly

Comment (bpostlethwaite) [inline Plotly.jl/url]: ah, yes strange, it has been named "Plotly.jl" at https://github.com/bpostlethwaite/METADATA.jl

**Comment by milktrader** [comment]
It might be easier for me to just merge this and really quick change it

**Comment by milktrader** [comment]
Yeah, let me try that

**Comment by milktrader** [comment]
Ok, should be fixed now. Still not sure what mechanism created  Plotly.jl/ instead of Plotly/ directory.

**Pull request merged**